src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,71 @@
 import React from 'react';
 import './CurrentUser.scss';
+import { loadUsers } from '../../api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type State = {
+  user: User | null,
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+interface Props {
+  userId: number,
+  clear: () => void,
+}
+
+export class CurrentUser extends React.Component<Props, State> {
+  state: State = {
+    user: null,
+  };
+
+  async componentDidMount() {
+    const user = await loadUsers(this.props.userId);
+
+    this.setState({
+      user,
+    });
```

Comment:
make a separate function and call it here. You use the same code here and in componentDidUpdate